grails-app/domain/org/pih/warehouse/invoice/Invoice.groovy

```diff
@@ -225,7 +231,7 @@ class Invoice implements Serializable {
             invoiceType: invoiceType?.code?.name(),
             hasPrepaymentInvoice: hasPrepaymentInvoice,
             isPrepaymentInvoice: isPrepaymentInvoice,
-            documents: documents,
+            documents: orderDocuments,
```

Comment:
LGTM, but this will probably need to change in the next ticket into `orderDocuments: orderDocuments`.

Reply:
OK